app/gui/view/component-browser/breadcrumbs/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "ensogl-breadcrumbs"
```

Comment:
The name suggests that this is a crate with general-purpose breadcrumbs component, but the location suggests that it is dedicated for IDE application.